client/src/main/java/org/hyperskill/database/clientside/Commands.java

```diff
@@ -0,0 +1,14 @@
+package org.hyperskill.database.clientside;
+
+import com.beust.jcommander.IParameterValidator;
+import com.beust.jcommander.ParameterException;
+
+public class Commands implements IParameterValidator {
```

Comment:
Better name it `CommandValidator`, cause just `Commands` is ambiguous
The name of a class should reflect on what it does